filters/tls/pass_client.go

```diff
@@ -0,0 +1,75 @@
+package tls
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"strings"
+
+	"github.com/zalando/skipper/filters"
+)
+
+type tlsSpec struct{}
+type tlsFilter struct{}
+
+func New() filters.Spec {
+	return &tlsSpec{}
+}
+
+func (*tlsSpec) Name() string {
+	return filters.TLSName
+}
+
+func (c *tlsSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
+	if len(args) != 0 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	return &tlsFilter{}, nil
+}
+
+const (
+	certSeparator           = ","
+	xForwardedTLSClientCert = "X-Forwarded-Tls-Client-Cert"
+)
+
+// sanitize the raw certificates, remove the useless data and make it http request compliant.
+func sanitize(cert []byte) string {
+	return strings.NewReplacer(
```

Comment:
strings.NewReplacer documented to be safe for concurrent use by multiple goroutines, maybe it makes sense to have one global instance

Reply:
makes sense, I just copied the code more or less as it was, but does not hurt to make it better :)

Reply:
done